addons/website_sale_wishlist/static/src/js/website_sale_wishlist_utils.js

```diff
@@ -56,10 +56,24 @@ function updateWishlistNavBar() {
     wishlistQuantity.classList.toggle('d-none', !wishlistProductIds.length);
 }
 
+function updateWishlistView() {
+    const wishlistProductIDs = getWishlistProductIds();
+    const wishlistEmpty = document.querySelector('#empty-wishlist-message');
+
+    if (wishlistProductIDs.length === 0) {
+        wishlistEmpty.classList.remove('d-none');
+        wishlistEmpty.classList.add('d-flex');
+    } else {
+        wishlistEmpty.classList.remove('d-flex');
+        wishlistEmpty.classList.add('d-none');
+    }
```

Comment:
You can simplify this using `toggle` 😉 

```suggestion
    wishlistEmpty.classList.toggle('d-none', !wishlistProductIDs.length === 0);
    wishlistEmpty.classList.toggle('d-flex', wishlistProductIDs.length === 0);
```